worldedit-core/src/main/java/com/sk89q/worldedit/function/block/BlockDistributionCounter.java

```diff
@@ -35,18 +38,28 @@
 public class BlockDistributionCounter implements RegionFunction {
 
     private final Extent extent;
+    private final Mask mask;
     private final boolean separateStates;
 
     private final List<Countable<BlockState>> distribution = new ArrayList<>();
     private final Map<BlockState, Countable<BlockState>> map = new HashMap<>();
 
     public BlockDistributionCounter(Extent extent, boolean separateStates) {
+        this(extent, null, separateStates);
+    }
+
+    public BlockDistributionCounter(Extent extent, @Nullable Mask mask, boolean separateStates) {
         this.extent = extent;
+        this.mask = mask == null ? Masks.alwaysTrue() : mask;
         this.separateStates = separateStates;
     }
 
     @Override
     public boolean apply(BlockVector3 position) throws WorldEditException {
+        if (!mask.test(position)) {
+            return true;
```

Comment:
This should likely return false. I doubt we use the value within WE, but generally RegionFunctions should only return true if "something was changed" (though in this case it's a counter rather than the world).